hr_expense_cancel/readme/DESCRIPTION.rst

```diff
@@ -1,3 +1 @@
-This module allows you to cancel and correct expenses. It adds a cancel button
-on the expense sheet that undo reconciliations and delete payments and journal
-entries.
+This module allows you to cancel and correct expenses. It adds a cancel button on the expense sheet that undo reconciliations and delete payments and journal entries.
```

Comment:
Why changing this without need? And the change is incorrect at RST syntax level

Reply:
Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)

Reply:
Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)